package.json

```diff
@@ -89,12 +98,68 @@
       "*/**/test/unit/**/*.test.js"
     ]
   },
-  "standard": {
-    "env": [
-      "jest"
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{js}": [
+      "npx prettier --write",
+      "npx eslint --fix"
+    ],
+    "*.{json,md,yml}": [
+      "npx prettier --write"
+    ]
+  },
+  "eslintConfig": {
+    "env": {
+      "node": true
+    },
+    "extends": [
+      "airbnb/base",
+      "plugin:jest/style",
+      "prettier"
+    ],
+    "ignorePatterns": [
+      "build/*"
     ],
-    "ignore": [
-      "test/**/*"
+    "parser": "@babel/eslint-parser",
+    "rules": {
+      "global-require": "off",
+      "no-console": "off",
+      "no-underscore-dangle": "off",
+      "no-use-before-define": [
+        "error",
+        {
+          "functions": false
+        }
+      ]
+    },
+    "overrides": [
+      {
+        "env": {
+          "jest": true
+        },
+        "files": [
+          "test/**/*.js"
+        ],
+        "plugins": [
+          "jest"
+        ],
+        "rules": {
+          "no-unused-vars": [
```

Comment:
Often we use mock factory pattern and so callbacks will often have unused variables.